src/main/java/org/sonarlint/intellij/analysis/DefaultClientInputFile.java

```diff
@@ -50,6 +59,21 @@
     return charset;
   }
 
+  @Override public InputStream inputStream() throws IOException {
+    if(doc == null) {
```

Comment:
A later commit will probably put a space in the `if(`, which is noise... Better fix it now ;-)